src/main/java/com/vso/model/service/login/LoginSession.java

```diff
@@ -15,17 +15,17 @@ public class LoginSession {
     public LoginSession(AuthenticationService authenticationService, AuthController authController) {
         this.authenticationService = authenticationService;
         this.authController = authController;
-    }
+    }}
 
-    public void initSession() {
-        while (true) {
-            authController.authenticateUser();
-            while (authenticationService.hasLoggedUser()) {
-                loggedUser = authenticationService.getLoggedUser();
-                loggedId = loggedUser.getId();
-                Login login = new Login(loggedUser);
-                authController.processLoggedUserOptions();
-            }
-        }
-    }
-}
\ No newline at end of file
+//    public void initSession() {
```

Comment:
Why this method is commented? Is't this the way how we get the ID of the logged user?